storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst

```diff
@@ -0,0 +1,39 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.13/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+Improvements and New Features
+================================================================================
+
+- :jirabug:`1548`: |Percona Xtrabackup| enables updating the
```

Comment:
Can we update https://www.percona.com/doc/percona-xtrabackup/2.4/xtrabackup_bin/lru_dump.html with new options, rename it to "buffer pool dump" and reference it from RN? Also xtrabackup option reference should be updated. And `innodb_buffer_pool_dump_now` became `--dump-innodb-buffer-pool`

Reply:
This update is tracked by https://jira.percona.com/browse/PXB-1736 I've added a note to that ticket. I will make another PR for this update which will contain a reference to documentation from the release notes. The documentation update will be available from the site at the same time as the release notes.

Reply:
Still correct parameter name is `--dump-innodb-buffer-pool`. Please also thank Marcelo Altmann (@altmannmarcelo ) for contribution.

Reply:
Updated as requested